### PR TITLE
Add method name and request url to the attachment name

### PR DESCRIPTION
**Pull request by baflQA**
### Context
Currently, each filename is just Request. When looking for a given request, I need to open the details of each file one by one, and check the request body. Having the method name and the URL in the file name will make it much easier to find the request I'm looking for

#### Checklist
- [x] [Sign Allure CLA][cla]
- [ ] Provide unit tests

[cla]: https://cla-assistant.io/accept/allure-framework/allure2

### CODE REVIEW

File: allure-httpclient/src/main/java/io/qameta/allure/httpclient/AllureHttpClientRequest.java [hunk 1]
```diff
@@ -55,26 +55,34 @@ public AllureHttpClientRequest(final AttachmentRenderer<AttachmentData> renderer
...
-        Stream.of(request.getAllHeaders())
-                .forEach(header -> builder.setHeader(header.getName(), header.getValue()));
+	@Override
+	public void process(final HttpRequest request,
+						final HttpContext context) throws IOException {
 
-        if (request instanceof HttpEntityEnclosingRequest) {
-            final HttpEntity entity = ((HttpEntityEnclosingRequest) request).getEntity();
+		final HttpRequestAttachment.Builder builder = create(getAttachmentName(request),
+				request.getRequestLine().getUri())
```
Comment (pazone) [inline allure-httpclient/src/main/java/io/qameta/allure/httpclient/AllureHttpClientRequest.java:68]: You need to fix indentations here

File: allure-httpclient/src/main/java/io/qameta/allure/httpclient/AllureHttpClientRequest.java [hunk 2]
```diff
@@ -55,26 +55,34 @@ public AllureHttpClientRequest(final AttachmentRenderer<AttachmentData> renderer
...
 
-            final ByteArrayOutputStream os = new ByteArrayOutputStream();
-            entity.writeTo(os);
+		Stream.of(request.getAllHeaders())
+				.forEach(header -> builder.setHeader(header.getName(), header.getValue()));
 
-            final String body = new String(os.toByteArray(), StandardCharsets.UTF_8);
-            builder.setBody(body);
-        }
+		if (request instanceof HttpEntityEnclosingRequest && ((HttpEntityEnclosingRequest) request)
```
Comment (pazone) [inline allure-httpclient/src/main/java/io/qameta/allure/httpclient/AllureHttpClientRequest.java:74]: This adds complexity => unit test is required

Comment (baflQA) [reply allure-httpclient/src/main/java/io/qameta/allure/httpclient/AllureHttpClientRequest.java]: Removed, there is another PR for this change.

File: allure-httpclient/src/main/java/io/qameta/allure/httpclient/AllureHttpClientRequest.java (hunk 1 above)
Comment (baflQA) [reply allure-httpclient/src/main/java/io/qameta/allure/httpclient/AllureHttpClientRequest.java]: Should be fixed now

File: allure-httpclient/src/main/java/io/qameta/allure/httpclient/AllureHttpClientRequest.java [hunk 3]
```diff
@@ -15,25 +15,20 @@
...
-import org.apache.http.protocol.HttpContext;
+import static io.qameta.allure.attachment.http.HttpRequestAttachment.Builder.create;
 
 import java.io.ByteArrayOutputStream;
 import java.io.IOException;
 import java.nio.charset.StandardCharsets;
 import java.util.stream.Stream;
 
-import static io.qameta.allure.attachment.http.HttpRequestAttachment.Builder.create;
+import io.qameta.allure.attachment.*;
```
Comment (baev) [inline allure-httpclient/src/main/java/io/qameta/allure/httpclient/AllureHttpClientRequest.java:25]: don't use wildcard imports

Comment (baflQA) [reply allure-httpclient/src/main/java/io/qameta/allure/httpclient/AllureHttpClientRequest.java]: Done

**Comment by baflQA** [comment]
@baev any other remarks? Or we can merge this one?

**Comment by baev** [comment]
@baflQA @pazone thanks!

**Comment by baflQA** [comment]
Hi, @baev can We count on a release sometime soon?

**Comment by baev** [comment]
sure

**Pull request merged**